contracts/JBTiered721Delegate.sol

```diff
@@ -28,7 +28,7 @@ import './structs/JBTiered721Flags.sol';
   Votes: A helper for voting balance snapshots.
   Ownable: Includes convenience functionality for checking a message sender's permissions before executing certain transactions.
 */
-contract JBTiered721Delegate is JB721Delegate, Ownable, IJBTiered721Delegate, IERC2981 {
```

Comment:
why move Ownable to the superclass? Can we keep this as JBOwnable here @xBA5ED ?

Reply:
@xBA5ED